extensions/ipynb/extension.webpack.config.js

```diff
@@ -8,13 +8,19 @@
 'use strict';
 
 const withDefaults = require('../shared.webpack.config');
+const path = require('path');
 
 module.exports = withDefaults({
 	context: __dirname,
 	entry: {
-		extension: './src/ipynbMain.ts',
+		['ipynbMain.node']: './src/ipynbMain.node.ts',
+		notebookSerializerWorker: './src/notebookSerializerWorker.ts',
```

Comment:
We need this notebookSerializerWorker entry as we need to compile a new output, just copying this will not work (as this file as additional imports).